### DISCUSSION TITLE
Unable to implement Line Height in Pango text format in vips

### DISCUSSION THREAD

**Discussion by abhilather** [libvips/libvips#3592] [Q&A]
I want to use line height in pango and create a text image using vips. 

I am passing the following text in pango format - 
```
<span foreground='#ffffff' font_desc='Inter 13' line_height='26' weight='700'>Just ₹1,39,999 or Exchange</span>
```

I have tried passing the line height with pixels, em, as percentage but nothing seems to work and the generated text image has edges closely bound to the text boundary.

Is there any built in method in libvips or any workaround to generate text images with line height?

**Comment by jcupitt** [comment]
Hi @abhilather,

It should work, have you seen:

https://docs.gtk.org/Pango/pango_markup.html

For example:

```
vips text y.png "<span line_height='10'>hello^Mworld</span>"
```

To generate:

![y](https://github.com/libvips/libvips/assets/580843/2926ae6e-1930-4d6e-91a0-6bd6fd7a8720)
